### PR TITLE
Handle page get errors

### PR DESCRIPTION
**Pull request by Dragnucs**
This is my attempt at handling page get errors.

PS: I am new to Rust and my proposition might not be the best.

### CODE REVIEW

File: src/page.rs [hunk 1]
```diff
@@ -14,12 +14,21 @@ pub struct Page {
...
-        let mut res = reqwest::get(url).unwrap();
         let mut body = String::new();
-        res.read_to_string(&mut body).unwrap();
+
+        match reqwest::get(url) {
+            Ok(mut res) => {
+                if res.status() == reqwest::StatusCode::OK {
+                    match res.text() {
+                        Ok(text) => body = text,
+                        Err(e) => println!("[error] {}: {}", url, e),
```
Comment (madeindjs) [inline src/page.rs:26]: You may prefer [`eprintln`](https://doc.rust-lang.org/std/macro.eprintln.html) for error logging.

```suggestion
                        Err(e) => eprintln!("[error] {}: {}", url, e),
```

File: src/page.rs [hunk 2]
```diff
@@ -14,12 +14,21 @@ pub struct Page {
...
+                    }
+                }
+            },
+            Err(e) => println!("[error] {}: {}", url, e),
```
Comment (madeindjs) [inline src/page.rs:30]: You may prefer [`eprintln`](https://doc.rust-lang.org/std/macro.eprintln.html) for error logging.

```suggestion
            Err(e) => eprintln!("[error] {}: {}", url, e),
```

**Review by madeindjs — CHANGES_REQUESTED** [review]
Thanks for PR :+1: 

This seem good, I just have a suggestion about `println`

**Comment by Dragnucs** [comment]
@madeindjs Thank you for the suggestion. In have applied it.

**Pull request merged**